AfricanReligions/localisation/afr_event_texts.csv

```diff
@@ -145,3 +145,95 @@ EVTDESC_AFRREL066;The sun lady Magec is beautiful like no other and full of bles
+EVTDESC_TOG_7530;Alkali is a bori from the Yan Riga court, a patron of the dyers and weavers. In a visit to the local market I consider that the best way to honor him is to get myself a new wardrobe of elegant tunics.;;;;;;;;;;;;;x
+EVTOPTA_TOG_7530;Only the finest fabric.;;;;;;;;;;;;;x
+EVTOPTB_TOG_7530;This is too ostentatious for me.;;;;;;;;;;;;;x
+EVTDESC_TOG_7531;Doguwa is a bori from the Babbaku court, a patron of wrestlers. She is a very complex being made of two halves that not always agree with each other. I am tempted to summon her presence but it could carry a great risk, for I don�t know which half will answer my call.;;;;;;;;;;;;;x
+EVTOPTA_TOG_7531;Dear mother, please come to me!;;;;;;;;;;;;;x
+EVTOPTB_TOG_7531;No, it�s too risky;;;;;;;;;;;;;x
+EVTDESC_TOG_7532;You feel the spirit take possession of your body and you start to sing her song �With her white cloth she has come�. This is the lovely mother of wrestlers, and she has come to give you her strength.;;;;;;;;;;;;;x
+EVTOPTA_TOG_7532;I feel it already!;;;;;;;;;;;;;x
+EVTDESC_TOG_7533;You feel the spirit take possession of your body and you start to sing her song �With her white cloth she has come�. However this is not the blessed half of Doguwa, but the wild one of the bush, and has decided to punish you for calling her name.;;;;;;;;;;;;;x
+EVTOPTA_TOG_7533;No, please. My eyes!;;;;;;;;;;;;;x
+EVTDESC_TOG_7534;Kulita is a bori from the Tan Garki court, a little drummer that marches with the armies. Inspired by him I have experimented with using young boys beating drums to mark the footsteps of my marching troops. This may be a real tactical advantage.;;;;;;;;;;;;;x
+EVTOPTA_TOG_7534;One, two, one, two;;;;;;;;;;;;;x
+EVTDESC_TOG_7535;As my illness worsens I have tried contacting with the bori spirits. Today my prayers have being answered as I felt how I was possessed by KaiKai, one of the Yayan Zanzanna children of the plague. The lady of the disease took it away, my pain is finally gone.;;;;;;;;;;;;;x
+EVTOPTA_TOG_7535;I must sacrifice two hens in her name.;;;;;;;;;;;;;x
+EVTDESC_TOG_7536;Ayu the manatee is one of the Tan Dowa wild bori, he drags his victims underwater and drowns them. Kadda the crocodile is a lot more direct, as he break them with his jaws. In a recent spirit possession I had the opportunity to choose which of these spirits to call.;;;;;;;;;;;;;x
+EVTOPTA_TOG_7536;The sneaky manatee.;;;;;;;;;;;;;x
+EVTOPTB_TOG_7536;The ferocious crocodile.;;;;;;;;;;;;;x
+EVTDESC_TOG_7537;Recently I have being practicing a lot of wrestling, as the giant �nti did. It not only has made me feel great but it seems like my new defined abdomen has caught the eye of my [From.GetHusbandWife].;;;;;;;;;;;;;x
```

Comment:
I checked this one, and `FROM` is just the same as `ROOT` here.  Can you go back through your events and check that the localization functions actually make sense?

Reply:
Nope they're the wrong way round. Should I be using PREV here?

Reply:
No, PREV doesn't make any sense here.  You have to set up the event so that FROM is the right person, like the lover and new friend events do.  You need to read about how scopes work on the wiki if you don't understand them.